CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -95,25 +95,53 @@ export default function Report({ reportID }) {
     },
     {
       header: "Duplicates",
-      accessorFn: (row) => row.TotalDuplicates.toString(),
+      accessorFn: (row) => row.TotalDuplicates,
+      cell: (info) => (
+        <span
+          style={{ color: "blue", textDecoration: "underline", cursor: "pointer" }}
+        >
+          {info.getValue().toString()}
+        </span>
+      ),
       id: "TotalDuplicates",
       footer: (props) => props.column.id,
     },
     {
       header: "Missing From CDC",
-      accessorFn: (row) => row.TotalMissingFromCDC.toString(),
+      accessorFn: (row) => row.TotalMissingFromCDC,
+      cell: (info) => (
+        <span
+          style={{ color: "blue", textDecoration: "underline", cursor: "pointer" }}
+        >
+          {info.getValue().toString()}
+        </span>
+      ),
       id: "TotalMissingFromCDC",
       footer: (props) => props.column.id,
     },
     {
       header: "Missing From State",
-      accessorFn: (row) => row.TotalMissingFromState.toString(),
+      accessorFn: (row) => row.TotalMissingFromState,
+      cell: (info) => (
+        <span
+          style={{ color: "blue", textDecoration: "underline", cursor: "pointer" }}
```

Comment:
Should use TailwindCSS styling instead of css as the rest of the app is made with tailwind. Tailwind equivalent is className='text-blue-500 cursor-pointer underline' instead of style={{ color: "blue", textDecoration: "underline", cursor: "pointer" }}